### DISCUSSION TITLE
MongoDB does not integrate

### DISCUSSION THREAD

**Discussion by Firewal7** [coroot/coroot#320] [Q&A]
![mongo](https://github.com/user-attachments/assets/e13b3682-bdff-486e-91a1-5b9463632474)
![mongo1](https://github.com/user-attachments/assets/d0cc9ead-24a6-4e5c-9c02-2948489b5ae8)

Hello everyone, I can't integrate MongoDB, I installed mongodb (helm upgrade --install db --set "metrics.enabled=true,architecture=replicaset" oci://registry-1.docker.io/bitnamicharts/mongodb), additionally launched mongo-agent in docker, connected to VictoriaMetrics, entered data into the integration configuration (data is correct), access to mongo on the Internet is open, but it does not integrate. Tell me the specific conditions for integration?

**Comment by Firewal7** [comment]
up

**Comment by T100D** [comment]
I have had this problem on a bare metal setup.

Setting the bindIp to 0.0.0.0 solved it in a situation without credentials made mongo monitoring possible with the coroot-cluster-agent
